test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
What about moving them to `devDependencies` like Polymer team does for wc.js https://github.com/webcomponents/webcomponentsjs/blob/master/bower.json